docs/self-managed/operational-guides/production-guide.md

```diff
@@ -0,0 +1,572 @@
+---
+id: production-guide
+title: "Helm Chart Production Guide"
+sidebar_label: "Helm Chart Production Guide"
+description: "Learn how to set up the helm chart in a production setting."
+---
+
+## Overview
+
+This guide provides a simplified and streamlined architecture for core Camunda 8 applications using the Camunda Helm Chart. This setup minimizes complexity while offering a reliable foundation for most production use cases.
+
+The goal of this guide is to give you a **scenario-based, production focused, step-by-step guide** for setting up the Camunda Helm Chart. By following this guide, you will be familiar with all of the necessary requirements for having a production ready Camunda Helm Chart.
+
+## Prerequisites
+
+Before proceeding with the setup, ensure the following requirements are met:
+
+- **Kubernetes Cluster**: A functioning Kubernetes cluster with kubectl access.
+- **Helm**: Helm CLI installed
+- **DNS Configuration**: Access to configure DNS for your domain to point to the Kubernetes cluster ingress.
+- **TLS Certificates**: Obtain valid X.509 certificates for your domain from a trusted Certificate Authority.
+- **External Dependencies**: Provision the following external dependencies:
+  - **Amazon Aurora PostgreSQL**: For persistent data storage required for the Web Modeler component.
+  - **Amazon OpenSearch**: For indexing and analytics.
+  - **AWS Simple Active Directory**: For authentication and authorization.
+- **NGINX Ingress Controller**: Ensure the NGINX ingress controller is set up in the cluster.
```

Comment:
dangerous territory. I don't know whether the helm chart was ever adjusted but afaik it's relying on the defaults of the [ingress-nginx](https://github.com/kubernetes/ingress-nginx) controller.

Just saying `NGINX Ingress Controller` will result in people wanting and using the [NGINX Ingress Controller](https://docs.nginx.com/nginx-ingress-controller/) by F5.

If people google for it 1 by 1, they will end up based on the naming at F5.

I'd consider the naming + consider a direct link to the project you mean.

Reply:
we have had incidents in the past as people were using the F5 nginx ingress controller as it doesn't work out of the box with the helm chart, especially around gRPC.

Reply:
naming changed and link provided. Thanks.